### DISCUSSION TITLE
After your note today I tried to install Git Backup, I got an error

### DISCUSSION THREAD

**Discussion by jaxxtheone** [Guilouz/Creality-Helper-Script-Wiki#519] [Issues]
![Screenshot 2024-04-11 115041](https://github.com/Guilouz/Creality-K1-Series/assets/26436482/5c1cf26e-44bc-49e3-aa98-fe72b86a338b)

**Comment by jaxxtheone** [comment]
After that, I manually created a file "git-backup" folder, at the right place, but it didn't change anything, with the same error as above.

**Reply by Guilouz** [reply]
Remove and reinstall script, it's looks like you have missing files on your system.

**Reply by jaxxtheone** [reply]
I tried to remove the item (16). when I try "information" it marks it as installed. I tried to reinstall and got the message that it's already installed. Do you mean unroot and root it again?

**Reply by pruinis** [reply]
Hello
I also can't remove "git-backup". It marks it as installed.

**Reply by Jonweb08** [reply]
me too

**Comment by Guilouz** [comment]
Fixed in last release.
